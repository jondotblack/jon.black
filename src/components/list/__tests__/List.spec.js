import React from 'react';
import renderer from 'react-test-renderer';

import { List } from '../index';

describe('List', () => {
  it('should render correctly with all props', () => {
    const tree = renderer
      .create(
        <List>
          <List.Item>One</List.Item>
          <List.Item>Two</List.Item>
          <List.Item>Three</List.Item>
        </List>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
