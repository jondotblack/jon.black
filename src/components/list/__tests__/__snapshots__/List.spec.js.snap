// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List should render correctly with all props 1`] = `
.emotion-6 {
  border-top: 1px solid rgba(64,65,59,0.3);
}

.emotion-0 {
  padding-top: 25px;
  padding-bottom: 20px;
  font-size: 40px;
  line-height: 1.2;
  border-bottom: 1px solid rgba(64,65,59,0.3);
}

@media (max-width:992px) {
  .emotion-0 {
    font-size: 28px;
    line-height: 34px;
  }
}

.emotion-0 a {
  display: block;
  padding-right: 60px;
  background: url('/static/svg/icon_external_link.svg') 100% 50% no-repeat;
}

.emotion-0 p:last-of-type {
  margin-bottom: 0;
}

.emotion-0 p a {
  display: inline-block;
  background: none;
}

<div
  className="emotion-6 emotion-7"
>
  <div
    className="emotion-0 emotion-1"
  >
    One
  </div>
  <div
    className="emotion-0 emotion-1"
  >
    Two
  </div>
  <div
    className="emotion-0 emotion-1"
  >
    Three
  </div>
</div>
`;
